bundles/org.openhab.binding.tivo/src/main/java/org/openhab/binding/tivo/handler/TiVoHandler.java

```diff
@@ -0,0 +1,364 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tivo.handler;
```

Comment:
Any reason this is not in the `internal` package? Same for the constants.

Reply:
fixed